src/app/backend/replicationcontrollerdetail.go

```diff
@@ -322,6 +317,37 @@ func getInternalEndpoint(serviceName, namespace string, ports []api.ServicePort)
 	}
 }
 
+// Returns array of external endpoints for a replication controller.
+func getExternalEndpoints(replicationController api.ReplicationController, pods []api.Pod,
+	service api.Service) []Endpoint {
+	var externalEndpoints []Endpoint
+	for _, externalIP := range service.Status.LoadBalancer.Ingress {
+		externalEndpoints = append(externalEndpoints, getExternalEndpoint(externalIP, service.Spec.Ports))
+	}
+	// For type node ports or if load balancer is unavailable display node ports.
+	if (len(externalEndpoints) <= 0 && service.Spec.Type == api.ServiceTypeLoadBalancer) ||
```

Comment:
My mistake, I always tend to use `=<` instead of `==` but in this case it makes no sense. Fixed.